### PR TITLE
IDEMPIERE-4365:[2pack] pack out translate for MTable MColumn

### PR DESCRIPTION
**Pull request by hieplq**
fixed comment from @hengsin 
update pull request by conflict to master

### CODE REVIEW

File: org.adempiere.pipo.handlers/src/org/adempiere/pipo2/handler/ColumnElementHandler.java [hunk 1]
```diff
@@ -348,6 +349,14 @@ public void create(PIPOContext ctx, TransformerHandler document)
 		addTypeName(atts, "table");
 		document.startElement("", "", I_AD_Column.Table_Name, atts);
 		createColumnBinding(ctx, document, m_Column);
+		
+		ctx.ctx.put("Table_Name",X_AD_Column.Table_Name);
+		try {
+			new CommonTranslationHandler().packOut(ctx.packOut, document, null, AD_Column_ID);
+		} catch(Exception e) {
+			if (log.isLoggable(Level.INFO)) log.log(Level.INFO, e.getMessage());
```
Comment (hengsin) [inline org.adempiere.pipo.handlers/src/org/adempiere/pipo2/handler/ColumnElementHandler.java:357]: Should be log.log(Level.INFO, e.getMessage(), e). As I've mention before, StackTrace matter. Same goes for the log below

**Comment by hengsin** [comment]
Hi @hieplq , I tested this and while it is working as expected, I'm wondering what's the use case for this when translation of column and table is usually overwritten by translation from ad_element. Can you clarify on this.

**Comment by hieplq** [comment]
has some reason for this path
1. for consistent to method export record as zip file
2. for consistent to method packout with type = "Data Single"
3. for consistent with field
4. not much but sometime i reuse core element for new customize column and it has new meaning. i can maintain translate at filed level, but why not column level

**Comment by globalqss** [comment]
I was thinking similar as Heng Sin, IMO Column Translation is something that we could simply remove from the system, from what I know it doesn't do any job, and just causes extra space in translations and in database.

**Comment by hieplq** [comment]
it's fine for me to remove column translate
but in case not yet remove it then this PR still helpful for consistent 2pack

**Comment by hengsin** [comment]
hi @hieplq ,

how do you maintain translation at column level ? there's no field equivalent of iscentralizedmaintain flag and sync terminology doesn't check column_trl.istranslate either.

**Comment by globalqss** [comment]
I would suggest instead of this commit, to deprecate the AD_Table_Trl and  AD_Column_Trl tables, AFAIK they have no use and is just consuming space in every translation, and confusing people if they think it requires translation.

**Comment by nmicoud** [comment]
Hello,
AD_Table_Trl is used on WAcctViewer and I think also on the FactAcctReset process (there, you can select a table).
Having it translated is better. So I don't think is a good idea to deprecate it.

FWIW, I'm using AD_Column_Trl on specific windows, to allow users to select a column to setup some specific stuff. Here again, having items translated is better, but I can change that to a reference.

**Comment by globalqss** [comment]
Ah - you're right @nmicoud - I see AD_Table_ID element is used in 72 tables and 10 process parameters.

About AD_Column_ID is being used in some user tables/windows, like AD_ChangeLog, AD_ImpFormat_Row, AD_PrintFormatItem, AD_WF_NextCondition, AD_WF_Node, C_AcctSchema_Element

So, I think my proposal doesn't make sense - it has some value to have the translation there, or we would need to change those references to use element translation, which seems a lot more difficult.

**Comment by globalqss** [comment]
Closing pull request after 38 days of inactivity - please feel free to create a new one or reopen with the comments solved.

After re-reviewing this I think the table AD_Column_Trl can be simply replaced by a view, as explained by Heng Sin above it's always getting the value from AD_Element_Trl.

**Pull request closed without merging**